astropy/wcs/tests/test_utils.py

```diff
@@ -257,6 +259,83 @@ def identify_offset(wcs):
         utils.wcs_to_celestial_frame(mywcs)
 
 
+def test_celestial_frame_to_wcs():
+
+    from ...coordinates import ICRS, FK5, FK4, FK4NoETerms, Galactic, BaseCoordinateFrame
```

Comment:
I thought for tests we generally put all imports at the top.

Reply:
This may have been to prevent circular imports, but I'll double check and will put a comment if so

Reply:
How can there be circular imports in tests?